File: QuickTest.Application/Users/Login/Props/AuthResponseDto.cs
namespace QuickTest.Application.Users.Login.Props;

public sealed record AuthResponseDto
{
    public int UserId { get; init; }
    public bool IsAuthSuccessful { get; init; }
    public string? ErrorMessage { get; init; }
    public string? Token { get; init; }
    public int SchoolId { get; set; }
}

File: Web/src/app/core/main/models/authResponseDto.ts
export interface AuthResponseDto {
  isAuthSuccessful: boolean,
  errorMessage: string,
  token: string,
  userId: string,
  schoolId: number
}

export interface UserInfoResponse {
  userId: string,
  firstName: string,
  lastName: string,
  email: string,
  userName: string
}